usr/share/rear/lib/bootloader-functions.sh

```diff
@@ -782,6 +782,34 @@ function make_pxelinux_config {
     echo "    append initrd=$PXE_INITRD root=/dev/ram0 vga=normal rw $KERNEL_CMDLINE $PXE_RECOVER_MODE"
     echo "say ----------------------------------------------------------"
 
+    # start with optional rear http entry if specified
+    if [[ ! -z $PXE_HTTP_URL ]] ; then    
```

Comment:
Because it introduces the new user config variable PXE_HTTP_URL
this variable must be described in usr/share/rear/conf/default.conf
e.g. something like
```
# Optional HTTP download source for PXE (URL style)
# for example PXE_HTTP_URL="http://pxe-over-http-srv:7777"
PXE_HTTP_URL=
```